src/libambit/sbem0102.c

```diff
@@ -173,10 +173,9 @@ int libambit_sbem0102_command_request(libambit_sbem0102_t *object, uint16_t comm
                     }
                 }
             }
-
-            ret = 0;
         }
 
+        ret = 0;
```

Comment:
Is this a cosmetic change or does it have an actual reason to move the initialization of "ret" here and below??

Reply:
It refers to my commit fce6ebb. If we don't do this, a request without response data is seen as invalid whatever request return status.
With log_synced request, I don't expect any return data, but the function "fails" while it shouldn't (return status is good and logs are synced on the watch)